homeassistant/components/openuv/strings.json

```
@@ -14,6 +14,9 @@
     "error": {
       "identifier_exists": "Coordinates already registered",
       "invalid_api_key": "Invalid API key"
```

One other latest standard is to use references for this too.
```suggestion
      "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]"
```